iOS/RNIntercom.xcodeproj/project.pbxproj

```diff
@@ -222,6 +222,7 @@
 			buildSettings = {
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
+					"~/Documents/Intercom",
```

Comment:
Why do you think that ? 🤔

Reply:
because is coupling a configuration with a location in your machine, and is not documented and now is just there as part of the eternal history.

Btw, Still failing to find that damn Intercom/Intercom.h
(Grabbing my knees and crying, three days and no progress)